### PR TITLE
PRO-1492: Replace Instabug

### PR DESCRIPTION
**Pull request by lalitkumarrock**
https://linear.app/pillarproject/issue/PRO-1492/pillar-wallet-replace-instabug

## Description

- Replaced Instabug functionality to Email Linking

## How Has This Been Tested?

- Tested on Android device

## Screenshots (if appropriate):
https://github.com/pillarwallet/pillarwallet/assets/82652040/00b89351-cba1-4c54-b932-9dddef830f48

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/constants/appConstants.js [hunk 1]
```diff
@@ -21,3 +21,4 @@
 export const IS_APP_VERSION_V3 = 'isAppVersionV3';
 export const API_REQUEST_TIMEOUT = 10000;
 export const DEFAULT_AXIOS_REQUEST_CONFIG = { timeout: API_REQUEST_TIMEOUT };
+export const EMAIL = 'support@pillarproject.io';
```
Comment (IAmKio) [inline src/constants/appConstants.js:24]: We should probably read this value from Firebase Remote Config?

File: ios/pillarwallet/Info.plist [hunk 2]
```diff
@@ -79,6 +79,14 @@
...
 		<string>PillarIcons.ttf</string>
 	</array>
+  <key>LSApplicationQueriesSchemes</key>
+  <array>
+      <string>mailto</string>
+      <string>airmail</string>
+      <string>ms-outlook</string>
+      <string>googlegmail</string>
+      <string>ymail</string>
+  </array>
```
Comment (IAmKio) [inline ios/pillarwallet/Info.plist:89]: Are all of these array items supposed to be here?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few queries but other than that - it LGTM

File: src/constants/appConstants.js (hunk 1 above)
Comment (lalitkumarrock) [reply src/constants/appConstants.js:24]: Okay, what is key?

File: ios/pillarwallet/Info.plist (hunk 2 above)
Comment (lalitkumarrock) [reply ios/pillarwallet/Info.plist:89]: Here all items are not mandatory. it's the app through which we can send emails. I can change it according to your expectation.

File: src/constants/appConstants.js (hunk 1 above)
Comment (IAmKio) [reply src/constants/appConstants.js:24]: `app_support_email` - currently set to support@pillarproject.io by default.

File: ios/pillarwallet/Info.plist (hunk 2 above)
Comment (lalitkumarrock) [reply ios/pillarwallet/Info.plist]: Removed it! mailto and whatsapp already exist in info.plist

File: src/constants/appConstants.js (hunk 1 above)
Comment (lalitkumarrock) [reply src/constants/appConstants.js]: Implemented it!

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**